Packs/Elasticsearch/Integrations/Elasticsearch_v2/README.md

```diff
@@ -132,12 +136,12 @@ Returns the schema of the index to fetch from. This commmand should be used for
 
 #### Input
 
-There are no input arguments for this command.
+| **Argument Name** | **Description** | **Required** |
```

Comment:
why include an empty table? Recommend just saying (no bullet)
There are no input arguments for this command.